docs/user/nifi/examples.rst

```diff
@@ -0,0 +1,55 @@
+Examples
+--------
+
+Ingest a GeoMesa Kafka Topic into HBase
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A common use case is to store the most recent state of an object in a GeoMesa Kafka data store to drive a live map
+view, but to persist every state of the object into long-term storage in HBase or Accumulo for history and
+analytics. This example will show how to pull data off a GeoMesa Kafka topic, and persist it in HBase with
+minimal configuration.
+
+Install the GeoMesa NARs
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Follow the steps under :ref:`nifi_install` to install the GeoMesa processors in your NiFi instance. This
+tutorial requires the GeoMesa Kafka and HBase NARs, as well as the standard GeoMesa service NARs.
+
+Add and Configure the Processors
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The first step is to add two processors to your flow, a ``GetGeoMesaKafkaRecord`` processor and an
+``AvroToPutGeoMesaHBase`` processor. Connect the output of the ``Get`` processor to the input of the ``Put``
+processor, and auto-terminate all the other connections:
+
+.. image:: /user/_static/img/nifi-kafka-to-hbase-flow.png
+   :align: center
+
+Next, create a new controller service by clicking 'Configure' on the NiFi flow, then going to the Controller
+Services tab and clicking the ``+`` button, then selecting the ``GeoAvroRecordSetWriterFactory``. The
+``GeoAvroRecordSetWriterFactory`` can be left with its default configuration, which is to use NiFi expressions.
```

Comment:
Should we mention enabling the GeoAvroRecordSetWriterFactory?  Or is that obvious?

Reply:
added it to the line at the bottom about enabling processors